Woof/WoofTests/Helper/DataTransformerTests/FiveStarRatingTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class FiveStarRatingTests: XCTestCase {
+    func testFiveStarRatingReturnsCaseUnavailableWhenNegativeValueProvided() {
```

Comment:
nit:
I can understand from the name of the class what will be tested here, why do you write the same FiveStarRating in each test func?
```suggestion
    func testReturnsCaseUnavailableWhenNegativeValueProvided() {
```

Reply:
Ok, fixed.